testSetup.js

```diff
@@ -2,6 +2,8 @@
 import 'jest-axe/extend-expect';
 import './app/assets/javascripts/lib/xss';
 
+global.setImmediate = global.setTimeout;
```

Comment:
Seems to be an issue with `setImmediate` and jest v27 https://github.com/facebook/jest/issues/11539#issuecomment-865854930